thoth-kafka-goodies/src/main/java/fr/maif/kafka/consumer/ResilientKafkaConsumer.java

```diff
@@ -230,7 +376,7 @@ protected String name() {
 
 
     protected Logger logger() {
-        return log;
+        return LoggerFactory.getLogger(ResilientKafkaConsumer.class);
```

Comment:
I think a private static final class attribute would be better that to instantiate a new Logger at each call